src/test/java/org/aksw/iguana/cc/storage/impl/TriplestoreStorageTest.java

```diff
@@ -26,12 +20,13 @@
  * @author f.conrads
  *
  */
+@Disabled("TriplestoreStorage hasn't been looked at in a while and the test is outdated.")
```

Comment:
should be re-enabled. For testing, use both TriplestoreStorage and RDFStorage and check that both contain the same graph. 

Query to retrieve the whole graph:
```SPARQL
construct {?x ?y ?z } where {?x ?y ?z}
```

There might also be a even simpler way with jena. 

We produce not blank nodes, right? Then checking for equality should be trivial. Otherwise, the models/graphs should be isomorphic. jena has methods for that.

Reply:
I did a little bit differnt now. I'm basically just checking, if the sent contains the correct data.